apps/app/.env.example

```
@@ -52,5 +52,8 @@ NEXT_PUBLIC_SKIP_DRIVER_VERIFICATION=true
 NEXT_PUBLIC_ENABLE_HOME_DASHBOARDS=false
 
 # ─── OpenAPI ─────────────────────────────────────────────────────
-OPENAPI_URL=https://coordinador-dev.mintral.cl
+OPENAPI_URL=https://your-openapi-host.com
 OPENAPI_TOKEN=your_api_token_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPENAPI_TOKEN key should go before the OPENAPI_URL key

(UnorderedKey)